### PR TITLE
implemented the_vital_message

### PR DESCRIPTION
**Pull request by ashwani010**

### CODE REVIEW

**Comment by ashwani010** [comment]
Please do review the code. Any kind of feedback is highly respected

File: docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py [hunk 1]
```diff
@@ -1 +1,47 @@
...
+#The 'time' module helps us to wait for some time
+
+
+import random
+import string
+import time
+print("VITAL MESSAGE")
+
+#Variable to store the difficulty i.e length of string
+D=0
```
Comment (freakboy3742) [inline docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py]: BASIC requires single letter variable names - we can do a lot better with Python :-)

File: docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#Loop until the right difficulty level is enterred
+while D<4 or D>10:
+    print("HOW DIFFICULT? (4-10)")
+    x=int(input())
+    D=x
+
+#Function to clear the screen
```
Comment (freakboy3742) [inline docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py]: Might want to explain *why* this clears the screen - although it's obvious to you and me, the audience for this is people who are just learning to code.

Comment (ashwani010) [reply docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py]: @freakboy3742 thank you for your valuable feedbacks and sorry for such mistakes.

**Pull request merged**